website/docs/r/workspace.html.markdown

```diff
@@ -9,6 +9,8 @@ description: |-
 
 Provides a workspace resource.
 
+~> **NOTE:** Setting the execution mode and agent pool affinity directly on the workspace has been deprecated in favor of using both [tfe_workspace_settings](workspace_settings) and [tfe_organization_default_settings](organization_default_settings). Use caution when unsetting `execution_mode` because the default value `"remote"` is no longer applied when the `execution_mode` is unset.
```

Comment:
As a user reading this, it's not clear to me what _will_ happen instead when I unset `execution_mode`. 🤔 Tho, as someone who's been in here reviewing recently, I think the answer is "it'll revert to the configured org default (whatever that happens to be) rather than a hardcoded default chosen by this provider".

Reply:
Oh wait, but actually, after reading the code: it won't revert to _anything,_ it'll just leave it what it was previously managed to (until something else changes it). And that's because, we _have_ to do it that way in order to continue supporting the deprecated attributes properly while allowing the new settings resource to take precedence when the deprecated attrs are absent. GOT IT.